Assets/Plugins/Source/Editor/Utility/GUIEditorUtility.cs

```diff
@@ -539,6 +540,11 @@ public static float MeasureLongestLabelWidth(List<string> labels)
             return MeasureLabelWidth(longestString);
         }
 
+        public static float MeasureLabelWidth(int characters)
+        {
+            return MeasureLabelWidth(new string('0', characters));
```

Comment:
Should probably use `M`, since typically (get it? typically?) this is the longest character in a typeset.

Reply:
Done. Marking conversation as resolved.